x/wasm/keeper/api.go

```diff
@@ -1,7 +1,10 @@
 package keeper
 
 import (
+	"fmt"
+
 	sdk "github.com/line/lbm-sdk/types"
+
 	wasmvm "github.com/line/wasmvm"
 	wasmvmtypes "github.com/line/wasmvm/types"
```

Comment:
```suggestion
	"fmt"

	sdk "github.com/line/lbm-sdk/types"
	wasmvm "github.com/line/wasmvm"
	wasmvmtypes "github.com/line/wasmvm/types"
```

The order of the imports is golang's native modules, external modules, github.com/line/xxx, github.com/line/wasmd/xxx.

Reply:
This isn't a big deal, though, so you can merge as is if other people think it's okay.